core/src/test/scala/nl/vroste/zio/kinesis/client/zionative/NativeConsumerTest.scala

```diff
@@ -575,9 +571,7 @@ object NativeConsumerTest extends ZIOSpecDefault {
               .flatMapPar(Int.MaxValue) { case (shard @ _, shardStream, checkpointer) =>
                 val out = shardStream
                   .tap(checkpointer.stage)
-                  .aggregateAsyncWithin(ZSink.collectAllN[Record[String]](200), Schedule.fixed(1.second))
```

Comment:
This causes interruption errors in `updateCheckpoint` for updating the checkpoint and release, when the shard stops processing.. It's a very complex ZStream operator, worth pinpointing the issue but out of scope for this PR.